web/js/panel.js

```diff
@@ -2257,7 +2257,7 @@ function $drawIcon($widget) {
             $("#panel-area>#" + $widget.id + "-overlay").css(ovlCSS);
         }
     } else {
-        jmri.log("ERROR: image not defined for " + $widget.widgetType + " " + $widget.id + ", state=" + $widget.state + ", occ=" + $widget.occupancystate);
+        console.error("ERROR: image not defined for " + $widget.widgetType + " " + $widget.id + ", state=" + $widget.state + ", occ=" + $widget.occupancystate);
```

Comment:
Note: You *cannot* rely on the console existing--there is no reason not to expect `console.error` to crash out the JavaScript completely on user's devices, so please revert this, or come up with a reliable alternative.

There is *zero* benefit to writing code to making unit testing easy if if crashes out entirely for users.